### ISSUE TITLE
bug on event form page with safari

### ISSUE THREAD

**Issue by benoitc** [erlef/website#245]
Safari doesn't display the calendar widgets on the start/end inputs. No error is displayed in the console.

**Comment by starbelly** [comment]
Will look into this.

**Comment by starbelly** [comment]
Safari and IE at this time do not support HTML5 calendar element 😞 . Best we can do for those cases without bringing in more JS is add place holders, which should exist anyway.

**Comment by starbelly** [comment]
Yeah, I guess we should bring in a small js lib to make a date picker available for browsers that do not support the html5 calendar element. As such, I labeled this as an enhancement.

**Comment by benoitc** [comment]
bug is still there. It's quite annoyiong to have to switch to another browser or wait to be on a computer to add an event. At least maybe adding the needed format in the fields would help?

**Comment by davydog187** [comment]
This might be a good one for me to take on

**Comment by starbelly** [comment]
@davydog187 re-assigned 👍
